### DISCUSSION TITLE
Is Query parameters need to encode when parameter type is array and items are define as objects?

### DISCUSSION THREAD

**Discussion by RSH17** [OAI/OpenAPI-Specification#3358] [Q&A] [answered]
when the query parameter is defined as type=array and its item as object. when executing, values get encoded. is that the behavior?

`"parameters": [
          {
            "name": "tags",
            "in": "query",
            "description": "Tags to filter by",
            "schema": {
              "type": "array",
              "items": {
                "type": "object"
              }
            }
          }
        ]`

result curl: curl -X 'GET' \
  'https://raw.githubusercontent.com/api/v3/pet/findByTags?tags=%7B%20%22R%22%3A%20100%2C%20%22G%22%3A%20200%2C%20%22B%22%3A%20150%20%7D&tags=%7B%20%22R%22%3A%20%22FF%22%2C%20%22G%22%3A%20%22GG%22%2C%20%22B%22%3A%20%22yu%22%20%7D' \
  -H 'accept: application/xml'

**Comment by handrews** [comment, marked as answer]
The Parameter Object does not say anything about encoding, so this behavior is surprising.  You should file an issue with whatever tool you are using, as we cannot help debug tooling problems here in the specification repo.

**Comment by handrews** [comment]
Closing as resolved due to lack of further questions - please comment if there are unresolved issues.
